packages/astro/src/types/public/config.ts

```diff
@@ -1858,9 +1858,6 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 			: Partial<Record<NormalizeLocales<NoInfer<TLocales>>, string>>;
 	};
 
-	/** ! WARNING: SUBJECT TO CHANGE */
-	db?: Config.Database;
-
```

Comment:
Is this a breaking change? I guess the warning is enough for that to be fine!

Reply:
Yeah that's what I thought! Technically breaking but given the comment, properly not subject to semver